lib/ansible/modules/cloud/cloudstack/cs_nic.py

```diff
@@ -192,6 +192,15 @@ def get_nic(self):
         if nics:
             self.nic = nics['nic'][0]
             return self.nic
+        self.result['changed'] = True
+        res = self.cs.addNicToVirtualMachine(**args)
+        if 'errortext' in res:
+            self.module.fail_json(msg="Failed: '%s'" % res['errortext'])
+
+        poll_async = self.module.params.get('poll_async')
+        if poll_async:
```

Comment:
If would just use polling without checking fro poll async because we rely on it, there is not really an option to not polling here.